website/settings/defaults.py

```diff
@@ -158,7 +158,8 @@ def parent_dir(path):
 SHORT_DOMAIN = 'osf.io'
 
 # TODO: Combine Python and JavaScript config
-COMMENT_MAXLENGTH = 500
+# Add 500 to account for variable length due to at mentions
+COMMENT_MAXLENGTH = 500 + 500
```

Comment:
Alright, so we have one layer of validation in JavaScript and another here. Can we get away with one? I guess not, since we need to validate API POSTs on the server-side, and we want the UI validation to be tighter (how does client-side validation work? How does it limit to 500 instead of 500 + 500?).